ttnn/cpp/ttnn/operations/uniform/uniform.hpp

```diff
@@ -0,0 +1,26 @@
+
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/decorators.hpp"
+#include "ttnn/operations/core/compute_kernel/compute_kernel_config.hpp"
+
+namespace ttnn::operations::uniform {
+
+struct Uniform {
+    static Tensor invoke(
+        const Tensor &input_tensor,
+        const float from,
+        const float to,
+        const std::optional<ttnn::MemoryConfig> &memory_config,
+        std::optional<const ttnn::DeviceComputeKernelConfig> compute_kernel_config);
```

Comment:
Please use `const std::optional<DeviceComputeKernelConfig>& compute_kernel_config` for consistency.
Also, check if can remove `ttnn::` here.

Reply:
updated.